src/DotVVM.Framework/Controls/GridViewValueColumn.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DotVVM.Framework.Binding;
+using DotVVM.Framework.Binding.Expressions;
+using DotVVM.Framework.Hosting;
+
+namespace DotVVM.Framework.Controls
+{
+    /// <summary>
+    /// A base class for GridView columns that present a value by wrapping a control.
+    /// </summary>
+    public abstract class GridViewValueColumn : GridViewColumn
```

Comment:
I'm not a fan of making the inheritance hierarchy deeper. In this case, it may lead to quite a bit of problems:
* What if I want a column that renders a control but does have the one distinctive "Value". For example combobox could be validated according to the `SelectedValue`, but the property probably should not be called `Value` for consistency with the original control.
* I may also want to have column with Value where validation does not make much sense. (example might be any non-trivial read-only component)
* I may already want to have a custom base class. "value column" may be one of many traits of that control, not the main super-class. 
* `ValueBinding` property does not have type specified (which is a problem for `GridViewCheckBoxColumn`)

I know they have something similar in BP, but we don't have to repeat every mistake here. Creating more classes named the same also won't probably help.